### ISSUE TITLE
[Bug]: Copy document tree (with inheritance) fails Pimcore 10.6/11

### ISSUE THREAD

**Issue by valentin-** [pimcore/pimcore#15428]
### Pimcore version

10.6.2

### Steps to reproduce

There's an error when copying document trees with inheritance. The server dies without giving a useful message (even in debug mode). Copying without inheritance works in 10.6

![image](https://github.com/pimcore/pimcore/assets/90033/20a0c5f7-3ed4-442f-b891-d52cfecef6b1)

I have a local setup of the Pimcore Demo. It worked fine with 10.5, after updating to 10.6 the same error occured.

There is also an error present in the current live demo (Pimcore 11).

### Expected / actual behavior

Copy without errors :)

**Comment by mattamon** [comment]
Hey @valentin- could you please also send show the details of the error message. This would help a lot to identify the problem.
Thank you!

**Comment by valentin-** [comment]
Hi @mattamon, as said above there is no useful error message.

It's easy reproducable in the Pimcore 10 demo with content.

1. Copy de 
![image](https://github.com/pimcore/pimcore/assets/90033/29c7176e-7817-4c30-9327-a4dcab4390bf)

2. Paste language recursive, updating references. Select fr
![image](https://github.com/pimcore/pimcore/assets/90033/1a1a0a7d-d146-44c8-9803-2f1dd0b09046)

3. "Error message", no more information using dev mode. No errors in dev.log
![image](https://github.com/pimcore/pimcore/assets/90033/1b3efef4-5492-4d04-a672-3bdc0449bab9)

I tried to debug it but couldn't find the main reason. I figured out that there was a problem copying the tree using pimcore_relations editables, but that wasn't the only issue.

My workaround was to disable the output of pimcore_areablock while the copy process is active. Is the document preview/renderer needed at this step?

**Comment by tommasoladinikumbe** [comment]
Hi, this seems to be caused by a wrong type hint (boolean instead of string) for the "language" parameter in the copyAsChild function (https://github.com/pimcore/pimcore/blob/11.x/models/Document/Service.php).
When the language is passed, it is converted to a boolean, causing an error when setting it as a property of the document.

**Comment by blankse** [comment]
@tommasoladinikumbe I created a PR https://github.com/pimcore/pimcore/pull/15440
The type was added in 11.0, so why did it not work in 10.6?

Edit: I debugged it. There is a endless loop with inherited editables.

**Comment by dvesh3** [comment]
> My workaround was to disable the output of pimcore_areablock while the copy process is active. Is the document preview/renderer needed at this step?

@valentin- thanks for the detailed steps. I could reproduce it on my end.

So disabling the output of `pimcore_areablock` fixed the bad gateway issue for you? my guess is that the render is called for checking the required editables and if this is the case then we might need to discuss if this is really required on copy. We'll take a look and thanks for the hint 👍

**Comment by valentin-** [comment]
@dvesh3 yes I just did a dirty, temporary workaround in `DefaultController::defaultAction` to hide the content while copying. 
It's ugly, just to prove that hiding the output "fixes" the issue.

```
public function defaultAction(Request $request)
{
    $hideContent = false;
    if(isset($_POST['sourceId'], $_POST['transactionId'])) {
        $hideContent = true;
    }

    return $this->render('default/default.html.twig', [
        'hide_content' => $hideContent,
    ]);
}
```

**Comment by kingjia90** [comment]
@valentin- I've found a potential solution. Could you please try this https://github.com/pimcore/pimcore/pull/15504?

**Comment by dvesh3** [comment]
Fixed by #15504
